.travis.yml

```diff
@@ -3,11 +3,16 @@ sudo: false
 
 matrix:
   include:
-    - go: 1.6
-    - go: 1.7
-    - go: 1.8
-    - go: 1.9
+    - go: "1.7"
+    - go: "1.8"
+    - go: "1.9"
+    - go: "1.10"
     - go: tip
 
+# accomodate forks
+before_install:
+  - mkdir -p $TRAVIS_BUILD_DIR $GOPATH/src/github.com/jhump
+  - test ! -d $GOPATH/src/github.com/jhump/protoreflect && mv $TRAVIS_BUILD_DIR $GOPATH/src/github.com/jhump/protoreflect || true
```

Comment:
This is throwing me for a curve ball. What is the purpose of this?

Reply:
Forked repos will be pulled to github.com/otheruser/protoreflect which so package imports will pull from the wrong location and fail with errors like this:
github.com/jhump/protoreflect (download)
desc/descriptor.go:17:2: use of internal package not allowed
desc/descriptor.go:18:2: use of internal package not allowed

This is similar to: https://github.com/cloudflare/cfssl/blob/master/.travis.yml

Reply:
Oh, I think I see. So when you use your fork, you are pulling it down and then importing it as `github.com/jhump/protoreflect` instead of as `github.com/prasek/protoreflect`? I guess I would have instead inspected you to have updated the imports in your fork to use `prasek` or to just have this line in your forked `Makefile` instead of this one.